tests/openvino/native/quantization/test_weights_compression.py

```diff
@@ -899,12 +899,55 @@ def test_compression_for_different_dtypes(activation_dtype, weight_dtype):
 )
 @pytest.mark.parametrize(
     ("compression_args", "multiplier_of_calls"),
-    (
-        (dict(mode=CompressWeightsMode.INT4_ASYM, ratio=1), 0),  # data-free, no reducers
-        (dict(mode=CompressWeightsMode.INT4_ASYM, ratio=0.5), 1),  # 1 reducer for mixed precision
-        (dict(mode=CompressWeightsMode.INT4_ASYM, ratio=1, awq=True), 2),  # mean & shape reducer for AWQ
-        (dict(mode=CompressWeightsMode.INT4_ASYM, ratio=0.5, awq=True), 3),  # 2 - for AWQ + 1 - for Mixed Precision
-    ),
+    [
+        ({"mode": CompressWeightsMode.INT4_ASYM, "ratio": 1}, 0),  # data-free, no reducers
+        ({"mode": CompressWeightsMode.INT4_ASYM, "ratio": 0.5}, 1),  # 1 reducer for mixed precision
```

Comment:
suppose the default one is not needed then

Reply:
done